app/assets/javascripts/snowcrash/pages/methodologies.js.coffee

```diff
@@ -1,3 +1,5 @@
+#= require snowcrash/pages/methodologies/chart
```

Comment:
Don't we normally include all files directly from the theme manifest (snowcrash.js)?